tests/ui/target-feature/wasm-relaxed-simd.rs

```diff
@@ -0,0 +1,10 @@
+//@ only-wasm32
+//@ compile-flags: -Ctarget-feature=+relaxed-simd --crate-type=lib
+//@ build-pass
+#![feature(stdarch_wasm_relaxed_simd)]
```

Comment:
This can only be removed after https://github.com/rust-lang/stdarch/pull/1494 is merged.

Reply:
Is this PR blocked on that then?

Reply:
I'm unsure of the exact process here.
rust-lang/stdarch#1494 can be merged anytime, but apparently it needs the exact Rust version where it becomes stable, see https://github.com/rust-lang/stdarch/pull/1494#discussion_r1378231374.

I'm happy to include an update to Stdarch in this PR if rust-lang/stdarch#1494 is merged first.

Reply:
I tentatively assigned Rust v1.81 to https://github.com/rust-lang/stdarch/pull/1494, so now it can actually be merged anytime.

Reply:
This PR now includes an update of Stdarch to https://github.com/rust-lang/stdarch/tree/7783b01f63d25c1220bbe9731ec14bd2811c22cc, which includes https://github.com/rust-lang/stdarch/pull/1494.